docs/reference/commandline/build.md

```diff
@@ -300,3 +301,11 @@ like `ENV` values do.
 
 For detailed information on using `ARG` and `ENV` instructions, see the
 [Dockerfile reference](../builder.md).
+
+### Specify isolation technology for container (--isolation)
```

Comment:
I would shy away from using the word "kernel" referring to the isolation technology on Windows. As in Hyper-V containers, the isolation is a Hypervisor, not really the kernel.